### PR TITLE
[v7r0] Minor fixes for ElasticSearchDB base class

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Core
FIX: Better logs and defaults for ElasticSearchDB base class

ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/ElasticSearchDB.py [hunk 1]
```diff
@@ -23,6 +24,9 @@
 from DIRAC.FrameworkSystem.Client.BundleDeliveryClient import BundleDeliveryClient
 
 
+LOG = gLogger.getSubLogger(__name__)
```
Comment (atsareg) [inline Core/Utilities/ElasticSearchDB.py]: We use all-capitals names for constants, not for objects. I think the name should be changed

Comment (andresailer) [reply Core/Utilities/ElasticSearchDB.py]: But its a constant object... and this was used also in other places.

Comment (atsareg) [reply Core/Utilities/ElasticSearchDB.py]: I mean this convention is for string or int/float constants not for objects. Constant object is a strange concept to me

Comment (fstagni) [reply Core/Utilities/ElasticSearchDB.py]: I don't mind either option, but I point out that this same line is also in use elsewhere.

Comment (atsareg) [reply Core/Utilities/ElasticSearchDB.py]: I would then change everywhere to subLogger or gSubLogger. Why are we defining conventions which we are diluting easily ?

Comment (fstagni) [reply Core/Utilities/ElasticSearchDB.py]: I have changed it to `sLog` (in several places)

Comment (chaen) [reply Core/Utilities/ElasticSearchDB.py]: What convention ? There's nothing in there as far as I can tell https://dirac.readthedocs.io/en/latest/DeveloperGuide/CodingConvention/index.html
And if you go that way, you give an argument why it should be `LOG`:
```
>>> class A(object):
...   pass
... 
>>> type(A) == type(int)
True
```

**Pull request merged**